Automattic/jetpack
# JS Components: Use clicks to summon IconTooltip

Follow-up on #26408

#### Changes proposed in this Pull Request:

Modifies IconTooltip to summon the Popover upon clicks instead of hovers. 

#### Other information:


#### Jetpack product discussion


See #26408.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:





Storyboard:
* Apply this change to your local checkout and spin up Storyboard
`cd projects/js-packages/storybook && pnpm run storybook:dev`
* Ensure that the IconTooltip component works as described.

Search Pricing Table:
* Apply this change on a Jetpack site without a Search subscription.
* Navigate to `/wp-admin/admin.php?page=jetpack-search&new_pricing_202208=1`.
* Ensure that clicking on the information icon opens a persistent popover.
* Ensure that any links within the popover is clickable.
